### ISSUE TITLE
Pam's Harvestcraft Interaction Issues (Thermos / Crucible Bug)

### ISSUE THREAD

**Issue by KaneHart** [CrucibleMC/Crucible#69]
With Pam's and Crucible we hit a few issues via interactions.

The one big one is eating base foods tends to not work when looking at a block or the ground. It ends up eating it / desyncing the inventory, but actually not consuming / filling the food bar.

The fix to this is looking into the sky.

Another issue we hit was reported by one of our users something about clicking the fruit off tree's might been with a torch that acts up. This one I have not tested out, but I thought I throw it out incase it's a known interaction issue.

**Comment by zeff9801** [comment]
I have the same issue with another food mod, I also found out that interact event .POST doesn't get fired but .START does

**Comment by KyberJeffHason** [comment]
I have the same issue, I'm coding my own mod and everything connected with Interacting(guarding with swords for example) are broken, PlayerItemUseEvent works incorrectly .STOP doesn't even run, but . START run twice.

**Comment by EverNife** [comment]
@KyberJeffHason can you give us a class example with the error for us to have a point to start lookin at?

As you have found a way to reproduce the error, might help us find something related ot it.

**Comment by zeff9801** [comment]
The event simply does not get called, happens with every mod, it works in thermos or other forks, so i think it's caused by one of the thermos patches juan made

**Comment by RehabCZ** [comment]
Yep this is a long living problem it was even discussed in multiple GTNH issues mostly related to HC
GTNewHorizons/GT-New-Horizons-Modpack#475
GTNewHorizons/GT-New-Horizons-Modpack#7410

I have to admit that i have not tried that with vanilla forge or even with thermos but i did notice that while playing GTNH on private server with crucible.

**Comment by KyberJeffHason** [comment]
Actually, some problems were my fault. But .Start still starts twice, for example with sword it runs when you have just started guarding and then it runs when you stop guarding.
